docs/docs/install/environment-variables.md

```diff
@@ -51,16 +51,13 @@ These environment variables are used by the `docker-compose.yml` file and do **N
 | `PORT`                  | Web Port              | `3000`  | web              |
 | `SERVER_PORT`           | Server Port           | `3001`  | server           |
 | `MICROSERVICES_PORT`    | Microservices Port    | `3002`  | microservices    |
-| `MACHINE_LEARNING_PORT` | Machine Learning Port | `3003`  | machine learning |
```

Comment:
This one is still used by the ML service. The UI setting points requests at the given URL, while this is the port the ML service actually listens to.